go_sequencer/txprocessor/txprocessor.go

```diff
@@ -1242,3 +1243,49 @@ func (txProcessor *TxProcessor) applyExit(coordIdxsMap map[common.TokenID]common
 
 	return exitAccount, false, nil
 }
+
+// computeEffectiveAmounts checks that the L1Tx data is correct
+func (txProcessor *TxProcessor) computeEffectiveAmounts(tx *common.L1Tx) {
+	tx.EffectiveAmount = tx.Amount
+	tx.EffectiveDepositAmount = tx.DepositAmount
+
+	if tx.Type == common.TxTypeCreateAccountDeposit {
+		return
+	}
+
```

Comment:
the `hermez-node` has a `CreateAccountDepositTransfer` case here. I'm wondering if we don't need it. The check is done by 
```go
if tx.ToIdx >= common.UserThreshold && tx.FromIdx == common.Idx(0) { 
// ...
}
```

Reply:
We don't need to check CreateAccountDepositTransfer.